server/src/main/kotlin/suwayomi/tachidesk/graphql/queries/UpdateQuery.kt

```diff
@@ -4,21 +4,18 @@ import org.kodein.di.DI
 import org.kodein.di.conf.global
 import org.kodein.di.instance
 import suwayomi.tachidesk.graphql.types.UpdateStatus
-import suwayomi.tachidesk.graphql.types.UpdateStatusType
 import suwayomi.tachidesk.manga.impl.update.IUpdater
-import suwayomi.tachidesk.manga.impl.update.JobStatus
 
 class UpdateQuery {
     private val updater by DI.global.instance<IUpdater>()
 
     fun updateStatus(): UpdateStatus {
-        val status = updater.status.value
-        return UpdateStatus(
-            isRunning = status.running,
-            pendingJobs = UpdateStatusType(status.statusMap[JobStatus.PENDING]?.map { it.id }.orEmpty()),
-            runningJobs = UpdateStatusType(status.statusMap[JobStatus.RUNNING]?.map { it.id }.orEmpty()),
-            completeJobs = UpdateStatusType(status.statusMap[JobStatus.COMPLETE]?.map { it.id }.orEmpty()),
-            failedJobs = UpdateStatusType(status.statusMap[JobStatus.FAILED]?.map { it.id }.orEmpty())
-        )
+        return UpdateStatus(updater.status.value)
+    }
+
+    data class LastUpdateTimestampPayload(val timestamp: Long)
```

Comment:
Eventually I will want to move this to a specific timestamp scalar, but since we are using Long and Int already for manga and chapters its fine.